docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc

```diff
@@ -2,7 +2,9 @@
 = Adding `container` components to a devfile
 
 [role="_abstract"]
-Use the `container` component type to define an image-based configuration of a container in a workspace. This `container` component type allows you to incorporate custom tools into your workspace. A devfile can contain one or more components of the `container` type. The `component name` attribute identifies each component, including the container.
+To incorporate custom tools into the workspace, you can define an image-based configuration of a container in a workspace. You can do this by using the `container` component type. 
```

Comment:
"you can" is superfluous here. *You can* put "you can" in front of pretty much any sentence :)